### ISSUE TITLE
ATAK crash due to NullPointerException on receiving message

### ISSUE THREAD

**Issue by a-f-G-U-C** [paulmandal/atak-forwarder#81]
The plugin is working fine on most devices I've tried it on, except on two older phones (a Samsung S6 and a no-name MT6737), where it crashes ATAK with a NullPointerException. The backtrace is:
```
05-15 09:59:04.571 32249 32249 E ACRA    : ACRA caught a NullPointerException for com.atakmap.app.civ
05-15 09:59:04.571 32249 32249 E ACRA    : java.lang.NullPointerException: Attempt to read from field 'boolean androidx.lifecycle.LiveData$ObserverWrapper.mActive' on a null object reference
05-15 09:59:04.571 32249 32249 E ACRA    :      at androidx.lifecycle.LiveData.considerNotify(LiveData.java:117)
05-15 09:59:04.571 32249 32249 E ACRA    :      at androidx.lifecycle.LiveData.dispatchingValue(LiveData.java:146)
05-15 09:59:04.571 32249 32249 E ACRA    :      at com.paulmandal.atak.forwarder.plugin.ui.viewmodels.StatusViewModel.onMessageAckNack(StatusViewModel.java:141)
05-15 09:59:04.571 32249 32249 E ACRA    :      at com.paulmandal.atak.forwarder.comm.meshtastic.MeshSender.lambda$handleMessageStatusChange$3$MeshSender(MeshSender.java:339)
05-15 09:59:04.571 32249 32249 E ACRA    :      at com.paulmandal.atak.forwarder.comm.meshtastic.-$$Lambda$MeshSender$rt(...)Y.run(lambda)
05-15 09:59:04.571 32249 32249 E ACRA    :      at android.os.Handler.handleCallback(Handler.java:739)
05-15 09:59:04.571 32249 32249 E ACRA    :      at android.os.Handler.dispatchMessage(Handler.java:95)
05-15 09:59:04.571 32249 32249 E ACRA    :      at android.os.Looper.loop(Looper.java:158)
```
The software configuration is:
- Android 6.0.0 (Mediatek) and 6.0.1 (Samsung) (factory install)
- Google Play Services 20.06.16 (installed as an update)
- Meshtastic app 1.2.31 (from github releases)
- ATAK-4.3.0.0-03e312ec-civ-debug.apk (from Drive)
- ATAK-Plugin-ATAK Forwarder-1.0.0-df0f358b-4.3.0-civ-debug.apk (from Drive)
- devices: firmware-tbeam-1.2.30.80e4bc6.bin (from github releases)

Both affected phones are older models running Android 6, and none of the newer phones seem to be affected by this problem.

**Comment by zgebones** [comment]
I'm getting the same error using Samsung S4s (Android 5.0.1) once I receive a response from a t-beam device ATAK crashes. I have tried ATAK and ATAK Forwarded version 4.4.0 and 4.5.1 with the same results. I have tried different version of Meshtastic t-beam firmware 1.2.50 and 1.2.52.

ATAKCRASH log:
"os.version":"3.4.0-4745659",
"android.release":"5.0.1",
"android.sdk":"21",
"device.model":"SPH-L720",
"device.manufacturer":"samsung",
"ACRA.version":"4.6.1",
"TAK.brand":"CIV",
"TAK.version":"4.4.0.12 (73de8851)",
"TAK.revision":"1637259897",
"TAK.plugin-api":"com.atakmap.app@4.4.0.CIV",
"TAK.error":"java.lang.NullPointerException: Attempt to read from field 'boolean androidx.lifecycle.LiveData$ObserverWrapper.mActive' on a null object reference",
"TAK.stackHash":"d58702e4",
"system.plugins": [],
"plugins":[{"plugin":"plugin.version.loaded.ATAK Forwarder","version":"1.0.3 (1) - [4.4.0]-1"}
]},

"os.version":"3.4.0-4745659",
"android.release":"5.0.1",
"android.sdk":"21",
"device.model":"SPH-L720",
"device.manufacturer":"samsung",
"ACRA.version":"4.6.1",
"TAK.brand":"CIV",
"TAK.version":"4.5.1.1 (2a1d81e3)",
"TAK.revision":"1641255505",
"TAK.plugin-api":"com.atakmap.app@4.5.1.CIV",
"TAK.error":"java.lang.NullPointerException: Attempt to read from field 'boolean androidx.lifecycle.LiveData$ObserverWrapper.mActive' on a null object reference",
"TAK.stackHash":"d58702e4",
"system.plugins": [],
"plugins":[{"plugin":"plugin.version.loaded.ATAK Forwarder","version":"1.0.4 (1) - [4.5.1]-1"}
]},
